src/openrct2/GameState.cpp

```diff
@@ -238,13 +240,32 @@ void GameState::UpdateLogic()
 
     if (network_get_mode() == NETWORK_MODE_SERVER)
     {
+        if (network_desync_debugging_enabled())
+        {
+            CreateStateSnapshot();
+        }
+
         // Send current tick out.
         network_send_tick();
     }
     else if (network_get_mode() == NETWORK_MODE_CLIENT)
     {
         // Check desync.
-        network_check_desynchronization();
+        bool desynced = network_check_desynchronization();
+        if (desynced)
+        {
+            log_verbose("Desynchronized");
```

Comment:
```suggestion
            log_verbose("Desynchronised");
```

Reply:
I'll remove that line as I've added better info in Network::CheckSRAND